src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static Map<Fruit, Integer> validRecords;
+    private static ReportServiceImpl reportService;
+
+    @BeforeClass
+    public static void setUp() {
+        reportService = new ReportServiceImpl();
+        validRecords = new HashMap<>();
+    }
+
+    @Test
+    public void makeReport_validData_Ok() {
+        for (int i = 1; i <= 3; i++) {
+            validRecords.put(new Fruit("F" + i), i);
+        }
```

Comment:
do we need loop that creates three questionably readable records?
i think it's better to create three normal-looking records manually